### PR TITLE
Set ios info plist unified 1.1.1

### PR DESCRIPTION
**Pull request by emeraldimp**
### New Pull Request Checklist

*Please mark the points which you did / accept.*

- [X] I read the [Step Development Guideline](https://github.com/bitrise-io/bitrise/blob/master/_docs/step-development-guideline.md)
- [X] I have a test for my Step, which can be run with `bitrise run test` (in the step's repository)
- [X] I did run `bitrise run audit-this-step` (in the step's repository - note, if you don't have this workflow in your `bitrise.yml`, [you can copy it from the step template](https://github.com/bitrise-steplib/step-template/blob/master/bitrise.yml).)
- [X] I read and accept the [Abandoned Step policy](https://github.com/bitrise-io/bitrise-steplib#abandoned-step-policy)

### CODE REVIEW

**Comment by viktorbenei** [comment]
Thanks for the PR!

Is there a reason why your step is a fork of https://github.com/teference/steps-set-ios-bundle-identifier ? I'm asking because in general we don't accept step forks as separate steps here, but your version seems to be sufficiently different, so it'd be better to unlink it from the upstream repo (https://github.com/teference/steps-set-ios-bundle-identifier). I guess you don't plan to send a PR there (?)

**Comment by emeraldimp** [comment]
@viktorbenei I will unhook it, I forked originally since it was based (heavily) on that step.

**Comment by emeraldimp** [comment]
Repo is unhooked

File: steps/set-ios-info-plist-unified/1.1.1/step.yml [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+source_code_url: https://github.com/mapletonhillmedia/steps-set-ios-info-plist
+support_url: https://github.com/mapletonhillmedia/steps-set-ios-info-plist/issues
+published_at: 2016-12-29T14:37:43.172584977-07:00
+source:
+  git: https://github.com/mapletonhillmedia/steps-set-ios-info-plist.git
+  commit: c727ccd56c882012d6d7013a469ab554ed82d971
+host_os_tags:
+- osx-10.10
+project_type_tags:
+- ios
```
Comment (viktorbenei) [inline steps/set-ios-info-plist-unified/1.1.1/step.yml]: can you please add the `macos` project type too? it should work for macOS app projects as well

File: steps/set-ios-info-plist-unified/1.1.1/step.yml [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+type_tags:
+- build
+- utils
+- xcode
+- xamarin
+- ios
+- info.plist
+run_if: .IsCI
```
Comment (viktorbenei) [inline steps/set-ios-info-plist-unified/1.1.1/step.yml]: With `run_if: .IsCI` the step won't run if you run it locally as part of a build, using the bitrise CLI.

I'm not sure what would be the best default value - do you think it's reasonable to assume that you don't want to run this step when you run a local build on your Mac? Or would you still want to run it?

File: steps/set-ios-info-plist-unified/1.1.1/step.yml [hunk 3]
```diff
@@ -0,0 +1,73 @@
...
+  commit: c727ccd56c882012d6d7013a469ab554ed82d971
+host_os_tags:
+- osx-10.10
+project_type_tags:
+- ios
+type_tags:
+- build
+- utils
+- xcode
+- xamarin
```
Comment (viktorbenei) [inline steps/set-ios-info-plist-unified/1.1.1/step.yml]: no need for xamarin here, it should be a project type tag

File: steps/set-ios-info-plist-unified/1.1.1/step.yml [hunk 4]
```diff
@@ -0,0 +1,73 @@
...
+- ios
```
Comment (viktorbenei) [inline steps/set-ios-info-plist-unified/1.1.1/step.yml]: same for ios, it should be a project type tag

File: steps/set-ios-info-plist-unified/1.1.1/step.yml (hunk 2 above)
Comment (emeraldimp) [inline steps/set-ios-info-plist-unified/1.1.1/step.yml]: Sure, I could see still wanting to run it.

**Comment by emeraldimp** [comment]
OK, I made the requested changes.

**Comment by viktorbenei** [comment]
Can you please do these changes in your step, and then release it as a new version? We had issues in the past when someone made the changes for an existing version, and then a new version did overwrite those.

**Comment by emeraldimp** [comment]
@viktorbenei For future contributors, that would've been good to know before I went to the trouble of updating the existing version.

**Comment by viktorbenei** [comment]
Sorry, I forgot to mention it.. My bad

**Pull request closed without merging**